### DISCUSSION TITLE
Open-VSX Decommissioning

### DISCUSSION THREAD

**Discussion by landryzs** [coder/code-server#6131] [General]
Hello!

Looks like [Open-VSX may be decommissioned](https://blogs.eclipse.org/post/john-kellerman/help-us-sustain-open-vsxorg) soon. What do next steps look like for Code-Server? Any plans to get Code-Marketplace to eventually reach parity with Open-VSX?

**Comment by code-asher** [comment]
I would like to bring it up to parity but do not have the time to dedicate to that at the moment (of course PRs are welcome).  To fully replace Open VSX I suppose we would need to host it as well.

**Reply by code-asher** [reply]
Looks like they are getting that group put together though: https://blogs.eclipse.org/post/john-kellerman/emerging-open-vsx-working-group

**Reply by landryzs** [reply]
Yeah I was thinking "worst case scenario" as if the working group didn't form for whatever reason. I'm assuming code-server would just switch back to using code-marketplace as the default?

**Reply by code-asher** [reply]
Really good question.  I am actually not sure we would want to get back into hosting extensions; definitely something we would need to discuss.
